pkgs/top-level/all-packages.nix

```diff
@@ -1431,7 +1431,7 @@ with pkgs;
 
   dosbox-staging = callPackage ../applications/emulators/dosbox-staging { };
 
-  duckstation = libsForQt5.callPackage ../applications/emulators/duckstation {};
+  duckstation = qt6Packages.callPackage ../applications/emulators/duckstation {};
```

Comment:
Remember me being more explicit next time.

I am trying to get rid of `rec` in my expressions, however many expressions that use `qt.*Packages` fail miserably.